File: netflipster/java/src/main/java/repository/FileDao.java
package repository;

import java.io.*;
import java.nio.file.*;

public class FileDao {
    public static String read(String id) {
        try {
            Path path = Paths.get("movies/" + id);
            return new String(Files.readAllBytes(path));
        } catch (IOException e) {
            return "No description available";
        }
    }
}

File: netflipster/DotNetCore/Netflipster/Repository/FileDao.cs
using System;
using System.IO;

namespace Netflipster.Repository
{
    public class FileDao
    {
        public static String Read(String id)
        {
            try
            {
                var fileLines = File.ReadAllLines("movies/" + id);
                var fileContent = string.Concat(fileLines);
                return fileContent;
            }
            catch (IOException e)
            {
                return "No description available";
            }
        }
    }
}
